node/node.go

```diff
@@ -426,6 +445,11 @@ func (n *Node) OnStart() error {
 		}
 	}
 
+	// Start background pruning
+	if err := n.pruner.Start(); err != nil {
+		return fmt.Errorf("failed to start background pruning routine: %w", err)
```

Comment:
Is it worth logging something here or no?

Reply:
What would you log - the error? If so, that's output here when running Comet as a standalone server: https://github.com/cometbft/cometbft/blob/24ebdd9df78d15828aa758e16a21d82fadcbb95b/cmd/cometbft/commands/run_node.go#L106-L108

The SDK would get the error when calling `Node.Start()` and they could choose what they'd want to do with it.